### PR TITLE
git-mv: add it translation

### PR DESCRIPTION
**Pull request by glenacota**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.it/common/git-mv.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Sposta o rinomina file e aggiorna l'indice Git.
+> Maggiori informazioni: <https://git-scm.com/docs/git-mv>.
+
+- Sposta i file nella repository e aggiungi l'operazione al commit successivo:
+
+`git mv {{percorso/al/file}} {{nuovo/percorso/al/file}}`
+
+- Rinomina i file e aggiungi l'operazione al commit successivo:
+
+`git mv {{percorso/al/file}} {{percorso/al/file_rinominato}}`
```
Comment (navarroaxel) [inline pages.it/common/git-mv.md:12]: This is not the 2nd example in the English page:
```suggestion
`git mv {{file}} {{file_rinominato}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @glenacota, thanks for your translation. Only 1 comment.

File: pages.it/common/git-mv.md (hunk 1 above)
Comment (glenacota) [reply pages.it/common/git-mv.md:12]: In my understanding, it actually is. The second example in the English page refers to a filename, which it's a path/to/file. You might think of a mapping `filename` -> `path/to/file` (i.e., `percorso/al/file`), and `new_filename` -> `path/to/renamed_file` (i.e., `percorso/al/file_rinominato`).

But if you think that is more clear stressing the name change only, I'll update my translation :)

Comment (navarroaxel) [reply pages.it/common/git-mv.md:12]: In Linux terms, the 1st and 2nd examples are identical, but I think the idea is only say `as user you can rename a file within the same folder`. You can await the review from another collaborator cc @sbrl @owenvoke

Comment (sbrl) [reply pages.it/common/git-mv.md:12]: Hrm, I don't know italian so I can't really comment here.

Comment (navarroaxel) [reply pages.it/common/git-mv.md:12]: me neither, but I think @glenacota is not talking about something specific to it language, right?

Comment (glenacota) [reply pages.it/common/git-mv.md:12]: > me neither, but I think @glenacota is not talking about something specific to it language, right?

exactly, I have the same doubt for the English version. But for the sake of this PR, I can translate the token literally ;)

**Review by schneiderl — APPROVED** [review]
LGTM

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**